java/manager-build.xml

```diff
@@ -298,135 +302,156 @@
+  <target name="restart-tomcat-container" depends="add-podman-connection" description="Restarts the tomcat process in a running pod container">
     <echo message="Launching Tomcat restart..."/>
-    <exec executable="kubectl">
-        <arg line="exec ${kubectl.args} -- rctomcat restart"/>
-    </exec>
+    <wrap-uyunictl argline="exec rctomcat restart"/>
+    <antcall target="remove-podman-connection"/>
   </target>
 
-  <target name="restart-taskomatic-kube" depends="warn-if-kubectl-not-installed" description="Restart the taskomatic process in a running kubernetes pod">
+  <target name="restart-taskomatic-container" depends="add-podman-connection" description="Restarts the taskomatic process in a pod container">
     <echo message="Launching Taskomatic restart..."/>
-    <exec executable="kubectl">
-      <arg line="exec ${kubectl.args} -- rctaskomatic restart"/>
-    </exec>
+    <wrap-uyunictl argline="exec rctaskomatic restart"/>
+    <antcall target="remove-podman-connection"/>
   </target>
 
-  <target name="deploy-restart-kube" depends="deploy-kube, restart-tomcat-kube, restart-taskomatic-kube" />
-
-  <target name="check-tomcat-has-opts">
-    <exec executable="kubectl" resultproperty="tomcat.has_opts_result" failonerror="false">
-      <arg line="exec ${kubectl.args} -- grep JAVA_OPTS /etc/sysconfig/tomcat"/>
-    </exec>
-    <condition property="tomcat.has_opts">
-      <equals arg1="${tomcat.has_opts_result}" arg2="0"/>
-    </condition>
+  <target name="deploy-restart-container">
+    <antcall target="deploy-container"/>
+    <antcall target="restart-tomcat-container"/>
+    <antcall target="restart-taskomatic-container"/>
   </target>
 
-  <target name="append-tomcat-debug-kube" depends="check-tomcat-has-opts" unless="tomcat.has_opts">
-    <exec executable="kubectl">
-        <arg value="exec"/>
-        <arg value="-n"/>
-        <arg value="${deploy.namespace}"/>
-        <arg value="${deploy.pod}"/>
-        <arg value="--"/>
-        <arg value="sh"/>
-        <arg value="-c"/>
-        <arg value="echo 'JAVA_OPTS=&quot;-Xdebug -Xrunjdwp:transport=dt_socket,address=\$HOSTNAME:8000,server=y,suspend=n&quot;' >>/etc/sysconfig/tomcat"/>
-    </exec>
-  </target>
 
-  <target name="replace-tomcat-debug-kube" depends="check-tomcat-has-opts" if="tomcat.has_opts">
-    <exec executable="kubectl">
-        <arg line="exec ${kubectl.args} -- sed 's/JAVA_OPTS=&quot;\([^&quot;]*\)&quot;/JAVA_OPTS=&quot;-Xdebug -Xrunjdwp:transport=dt_socket,address=\$HOSTNAME:8000,server=y,suspend=n \1&quot;/' -i /etc/sysconfig/tomcat"/>
+  <!-- wraps uyunictl command, executing the given argline and setting the CONTAINER_CONNECTION env var -->
+  <macrodef name="wrap-uyunictl">
+    <attribute name="argline"/>
+    <attribute name="failonerror" default="true"/>
+    <sequential>
+      <exec failonerror="@{failonerror}" executable="uyunictl">
+        <arg line="@{argline}"/>
+        <env key="CONTAINER_CONNECTION" value="${podman.connection.name}"/>
```

Comment:
This variable is not useful in the case of `kubectl` but can't harm if it's added unconditionnaly